src/Models/Role.php

```diff
@@ -52,7 +61,7 @@ public function permissions(): BelongsToMany
             config('permission.table_names.role_has_permissions'),
             'role_id',
             'permission_id'
-        );
+        )->select('id', 'name', 'guard_name');
```

Comment:
@drbyte I'm not sure about this, maybe is a breaking change if somebody already added custom fields and want to use them, but it get less data to memory(`updated_at`,`created_at`,`deleted_at`, etc properties)

Reply:
Yes I guess it could be a breaking change.
However, if they've added custom fields then they've probably also already overridden some of the other methods in this trait, so can easily override this one as well.

Perhaps this isn't necessary in this PR though? Maybe it should be separated for further exploration?  I don't like adding hundreds of configuration-settings but perhaps this is one place where it would be useful?

Reply:
>Perhaps this isn't necessary in this PR though?

Not necessary, i can remove that

Reply:
Reverted